### ISSUE TITLE
Adding notes section to the interface

### ISSUE THREAD

**Issue by abhi18av** [seqeralabs/nf-tower#260]
Hi team,

I know it might not make it into the project roadmap anytime soon (or at all), but I felt that it would be great if one could have notes associated with any specific pipeline.

**Comment by pditommaso** [comment]
Yeah, we even add a prototype but was not good enough. We'll do this after teams & role management.

**Comment by abhi18av** [comment]
Oh, I see. Good to know this is on the roadmap already  👍
